README.md

```diff
@@ -45,7 +48,26 @@ pip install morphology-workflows
 
 ## Usage
 
-This workflow is based on the ``luigi`` library but can be run via the command line interface:
+### Create inputs
+
+The inputs should consist in:
+
+* a directory containing the input morphologies.
+* a CSV file with the following columns:
+    1. ``index``: the row index.
+    2. ``morph_path``: the path to the morphology file.
+    3. ``morph_name``: the name of the morphology.
+    4. ``brain_region``: the brain region in which the morphology is located.
```

Comment:
this is optional, and any other column will be transferred without being affected, no?

Reply:
Ah I don't remember about this. I will check.

Reply:
I can't find where it is used (the example works if I remove it from the CSV file) so I guess it is optional.